src/components/List/ListItem/ListItem.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import bem from 'bem';
+import Text from '../../Text';
+import styles from './ListItem.scss';
+
+const { block, elem } = bem({
+    name: 'ListItem',
+    classnames: styles,
+    propsToMods: ['isDivided', 'isDividedList', 'isSelected', 'onClick']
+});
+
+const ListItem = props => {
+    const { children, isDivided, isDividedList, isSelected, onClick, ...rest } = props;
+    const customBlockMod = { clickable: typeof onClick === 'function' };
+
+    return (
+        <li {...rest} {...block(props, customBlockMod)}>
+            <div onClick={onClick} role="presentation" {...elem('container', props)}>
+                {React.Children.map(
+                    children,
+                    child => (typeof child === 'string' ? <Text>{child}</Text> : child)
+                )}
+            </div>
+        </li>
+    );
+};
+
+ListItem.displayName = 'ListItem';
+
+ListItem.propTypes = {
+    /** List Items */
+    children: PropTypes.node,
+    /** Formats this item as selected  */
+    isSelected: PropTypes.bool,
```

Comment:
Is this prop used both for 'selected' (result is selected) and 'active' (viewing details) state? Would it make sense to have a separate `isActive` prop? Even if by default both states would visually be the same, one may want to have it styled differently in a custom theme, and I'd say it's more explicitly differentiated

Reply:
See discussion in the design document. My understanding that for now we keep it with one prop. In the future if we have an actual use case of differentiating them, and we know what they really need to be called, we can add an other prop.